documents/entity/bootstrapper.go

```diff
@@ -63,11 +63,16 @@ func (Bootstrapper) Bootstrap(ctx map[string]interface{}) error {
 		return errors.New("entity relation service not initialised")
 	}
 
+	processor, _ := ctx[documents.BootstrappedAnchorProcessor].(documents.AnchorProcessor)
+	/*if !ok {
```

Comment:
I want to move the processor from documents.PostBootstrapper to documents.Bootstrapper